Cargo.toml

```
@@ -22,7 +22,7 @@ hashbrown = { version = "^0.9", optional = true }
 chrono = { version = "^0.4", optional = true }
 
 [dev-dependencies]
-deepsize_derive = { path = "deepsize_derive", version = "0.1.1" }
+deepsize_derive = { path = "deepsize_derive", version = "0.1.2" }
```

```suggestion
deepsize_derive = { path = "deepsize_derive" }
```

I *think* this should work. Cargo is a bit funky when it comes to dev-dependencies and publishing -- the publish crate doesn't have dev-dependencies section at all. Eg, if you run `cargo publish --dry-run`, you can see that `traget/package/deep-size-of-0.2.0/Cargo.toml` is an edited version, with dev deps excluded.